penpost/penpost-web
# Jk/feature/update info forms

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 

Adds a user landing page, that displays the user data (address and about me section) with the option to edit. It also displays user connections, conditionally rendering an option to make a connection. If a user does have a connection, it will display connection data (name, country, and connection about me), with the options to end the connection or send your connection a post card.

It also adds a form component where users are able to update their address and about me sections.

It includes a dependency for country/region selection.

Adds minor styling.

Issues closed #33 

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

No problems, but we will need to revisit the form submission and on page load useEffect to integrate graphQL requests.

> Describe your code changes in detail for reviewers.

No changes, on added components.

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

N/A

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Once router functionality has been fully integrated, user landing page should render with appropriate data displayed. Edit form should display on button click.

### Steps To Take
> Any questions and possible steps moving forward.

Take graphQL requests into account (will need to refactor later)
